### ISSUE TITLE
Darkmoon heroism proc is too low (currently takes 36 hits to proc heroism sometimes) vanilla sources say 10-11%

### ISSUE THREAD

**Issue by Mostprovidedlinks** [elysium-project/server#1699]
At the moment darkmoon heroisms proc chance is far less than 10%-11%

On Elysium It took an amazing 33-34 hits to proc it
https://youtu.be/y6N41yrrcZ8

And here it took 36 hits
https://youtu.be/iEShqqnDGLA

This is not a 10.4-11% proc chance

Stated 3 times here in vanilla wow
https://web.archive.org/web/20070905095129/http://thottbot.com:80/i19287

Comments

The Proc Rate
Score 2.3     Vote: [-] [+] by Vagrant123, 1.1 years ago 
Last edit: 1.1 years ago
On a sample test, out of 2402 hits, the card procced 276 times. That's roughly 11.5%. It's really helpful. 

276/2402 = 11.5%

 (more swings in this test)
  Report  ReplyProc rate
Score 3.9     Vote: [-] [+] by Vagrant123, 1.1 years ago
More data: 
775/7486 = 10.4%
It seems like the proc rate is 10%.

Score 1.4     Vote: [-] [+] by DoubleA224, 8.0 months ago
The proc on warlords is about 10%

Fix:

- since it was stated between 10.4-11.5%, change trinket to a 10.4% proc chance or 11% proc chance

**Comment by Flappynoodle** [comment]
fix please

**Comment by fidbugs** [comment]
Here is extensive research on subject of its proc chance.
#590 

TLDR - PPM should be 3. 

I suppose you tested on Elysium.

**Comment by Mostprovidedlinks** [comment]
Your video is from 4.0.1 and items could have changed by then due to the players spells. Quest items however don't change much. My comments show from 2006

**Comment by Mostprovidedlinks** [comment]
Your page is wrong. It does add up with this.

775/7486 = 10.4%

If we assume a rogue made a low ppm item reach 10%-11% then those figures would of been in the tens of thousands or twice to three times those hits from all the additional hits. 

276/2402 = 11.5%

Would most likely have 6000 hits in order to make a low ppm reach 11%

**Comment by Mostprovidedlinks** [comment]
I think we should just use 2006 results rather than 4.0.1

**Comment by Mostprovidedlinks** [comment]
Since if the proc was 3 times lower you would need 3 times more hits than shown here

**Comment by Mostprovidedlinks** [comment]
775/7486 = 10.4%

If we assume a rogue made a very low ppm item which gave much under 10% proc chance reach 10%-11% then those figures would of been in the tens of thousands or twice to three times those hits from all the additional hits.

276/2402 = 11.5%

Would most likely have 6000 hits in order to make a low ppm reach 11%

I think we should just use 2006 results rather than 4.0.1

Since if the proc was 3 times lower you would need 3 times more hits than shown here

**Comment by Mostprovidedlinks** [comment]
Whatever its ppm is it should be more.

I have swung 10 times with a 2 hand and not had it proc.
